modules/oc-mirror-migration-differences.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * oc-mirror-migration-v1-to-v2.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-migration-differences_{context}"]
+= Changes from oc-mirror plugin v1 to v2
+
+Before migrating from oc-mirror plugin v1 to v2, you can check differences between oc-mirror plugin v1 to v2 below:
+
+. *Explicit version selection*: Users must explicitly specify `--v2` when using `oc-mirror`. If no version is specified, v1 is executed by default. This behavior is expected to change in future releases (4.19 or later), where `--v2` will be the default.
+. *Command line changes*:   
+- `mirror` to `mirror`: Requires `--v2` and introduces `--workspace` in the command line.
+- `mirror to disk` & `disk to mirror`: Requires `--v2` and mandates `file://` for disk locations.
```

Comment:
Could we list the command : https://github.com/openshift/openshift-docs/pull/87754#discussion_r1946343005  here.